From 9852d7841c09f5ce6f166e5abf0b220c3e8ba96d Mon Sep 17 00:00:00 2001
From: Mike Fiedler <miketheman@gmail.com>
Date: Wed, 21 Jun 2023 07:18:19 -0400
Subject: [PATCH] fix: only emit license and readme for sdist

As the file contents are already being included in the wheel's
`dist-info` folder either alone or embedded into PKG-INFO, we don't need
to emit these ourselves.

Resolves #230

Signed-off-by: Mike Fiedler <miketheman@gmail.com>
---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 49d7c01..7fb2108 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -8,8 +8,8 @@ readme = "README.md"
 homepage = "https://pypi.org/project/pytest-socket/"
 repository = "https://github.com/miketheman/pytest-socket"
 include = [
-    { path = "LICENSE" },
-    { path = "README.md" },
+    { path = "LICENSE", format = "sdist" },
+    { path = "README.md", format = "sdist" },
     { path = "tests", format = "sdist" },
     { path = ".flake8", format = "sdist" },
 ]
